### PR TITLE
Add list-checked rule and corresponding test

### PR DESCRIPTION
**Pull request by magnuh**
Added list-checked class from here:
https://github.com/fabric-ds/css/blob/49b07c05686ef132b203026e941bf04dbab6fc7a/src/components/list.css

Question: Should we add fallbacks to all vars.

### CODE REVIEW

File: src/_rules/list.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  'padding-left': 'var(--w-space-3)',
+}));
+const checkmarkStyles = entriesToCss(Object.entries({
+  content: '""',
+  display: 'block',
+  position: 'absolute',
+  width: '16px',
+  height: '28px',
+  left: 0,
+  color: 'var(--w-list-color-icon-checked)',
```
Comment (magnuh) [inline src/_rules/list.js:14]:
```suggestion
  color: 'var(--w-list-color-icon-checked, hotpink)',
```
Should we be adding fallback to all variables?

Comment (pearofducks) [reply src/_rules/list.js:14]: No fallback needed in any of these rules IMO

File: src/_rules/list.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import { entriesToCss } from '@unocss/core';
+
+const listItemStyles = entriesToCss(Object.entries({
+  position: 'relative',
+  'padding-left': 'var(--w-space-3)',
```
Comment (pearofducks) [inline src/_rules/list.js:5]: I don't think the `space` values are ready for use?

Comment (magnuh) [reply src/_rules/list.js:5]: Perhaps not... 😄  How should we handle it for now?

Comment (pearofducks) [reply src/_rules/list.js:5]: I would just set it to static pixels for now like the heights and widths.

Comment (magnuh) [reply src/_rules/list.js:5]: Should we use "TODOs" in these places? Or would that make it too todo cluttered at this time? 😅

Comment (magnuh) [reply src/_rules/list.js]: Updated with todos.

**Review by pearofducks — APPROVED** [review]

**Comment by AnnaRybkina** [comment]
Great stuff! Merging 🚢

**Pull request merged**